app/Controllers/UserLogin.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Controllers;
+
+use CodeIgniter\API\ResponseTrait;
+use App\Models\UserModel;
+
+class UserLogin extends BaseController
+{
+    use ResponseTrait;
+
+    public function index()
```

Comment:
Use /** to write each function comments.

Reply:
done